CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 #
 #
 
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I'd suggest using `3.10..3.31` instead. CMake 3.10 was released in 2017.

Reply:
Okay, let's say 3.10 as in the PR in Nixpkgs.